### PR TITLE
bus-print-property: introduce BusPrintPropertyFlags

### PR DESCRIPTION
**Pull request by yuwata**
Follow-ups for 1ceaad69378272c64da4ecaab0d59ebb7a92ca0a.

### CODE REVIEW

File: src/machine/machinectl.c [hunk 1]
```diff
@@ -63,7 +63,7 @@
 
 static char **arg_property = NULL;
 static bool arg_all = false;
-static bool arg_value = false;
+static BusPrintPropertyFlags arg_print_flags = 0;
 static bool arg_full = false;
```
Comment (bluca) [inline src/machine/machinectl.c:67]: any reason to keep `arg_all` around in this one?

File: src/systemctl/systemctl.c [hunk 2]
```diff
@@ -75,7 +75,7 @@ bool arg_no_wtmp = false;
 bool arg_no_sync = false;
 bool arg_no_wall = false;
 bool arg_no_reload = false;
-bool arg_value = false;
```
Comment (bluca) [inline src/systemctl/systemctl.c:78]: same question about `arg_all` here

File: src/machine/machinectl.c (hunk 1 above)
Comment (yuwata) [reply src/machine/machinectl.c:67]: `arg_all` is also used for listing machines and images, and in those cases, `bus_print_property()` is not used. I think it is better to keep `arg_all` for readability. But if you do not like it, all `arg_all` can be replaced with `FLAGS_SET(arg_print_flags, BUS_PRINT_PROPERTY_SHOW_EMPTY)`.

File: src/systemctl/systemctl.c (hunk 2 above)
Comment (yuwata) [reply src/systemctl/systemctl.c:78]: Same for `systemctl`. `arg_all` is used at various places.

File: src/machine/machinectl.c (hunk 1 above)
Comment (bluca) [reply src/machine/machinectl.c:67]: it's ok as is, makes sense to me, thanks

**Review by bluca — APPROVED** [review]
LGTM

**Comment by poettering** [comment]
lgtm

**Pull request merged**